packages/studio-plugin/src/createStudioStylingPlugin.ts

```diff
@@ -2,7 +2,7 @@ import { PluginOption } from "vite";
 import upath from "upath";
 import { fileURLToPath } from "url";
 const currentDir = upath.dirname(fileURLToPath(import.meta.url));
-const packagesDir = upath.resolve(currentDir, "../..");
+const packagesDir = upath.resolve(currentDir, `..${upath.sep}..`);
```

Comment:
pretty sure `upath.sep` will always be `/` because it resolves to unix first, so you'd actually want `path.sep`. but I think this isn't necessary (and we can go back to what you had before) since we're already running this through `upath`. you can read their docs to double check though

Reply:
ok! I also made sure that we compare this unix `packageDir` to the unix version of `importer` in the studio import check.